index.js

```diff
@@ -311,6 +316,7 @@ module.exports = (command, args, options) => {
 	// eslint-disable-next-line promise/prefer-await-to-then
 	spawned.then = (onFulfilled, onRejected) => handlePromise().then(onFulfilled, onRejected);
 	spawned.catch = onRejected => handlePromise().catch(onRejected);
+	spawned.cancel = reason => processDone.cancel(reason);
```

Comment:
Maybe you should provide a default `reason` if none is passed, like `The command was canceled`